app/helpers/application_helper.rb

```diff
@@ -8,12 +8,12 @@ def active_page_class(name)
   # Wrap a link to the SearchWorks record for the given Catkey wrapped in the markup
   # necessary to be aligned with the content in the collapsible list sections
   def detail_link_to_searchworks(catkey)
-    content_tag(:div, class: 'row justify-content-center') do
-      content_tag(:div, class: 'col-5') do
+    content_tag(:div, class: 'row') do
+      content_tag(:div, class: 'col-11 offset-1 col-md-11 offset-md-2') do
```

Comment:
I know this is what was specified, but should the `col-md-11` be a `col-md-10`?

```suggestion
      content_tag(:div, class: 'col-11 offset-1 col-md-10 offset-md-2') do
```

Reply:
@jvine ?